package.json

```diff
@@ -29,7 +34,7 @@
   },
   "scripts": {
     "test": "mocha",
-    "postinstall": "make",
-    "start": "./bin/slackin $SLACK_SUBDOMAIN $SLACK_API_TOKEN"
+    "postinstall": "./node_modules/.bin/gulp",
```

Comment:
I thought NPM aliased the local binaries as globals for the script declarations?